lib/Doctrine/DBAL/Driver/Mysqli/MysqliConnection.php

```diff
@@ -63,6 +63,19 @@ public function __construct(array $params, $username, $password, array $driverOp
 
         $this->_conn = mysqli_init();
 
+        if (isset($driverOptions['ssl_key']) &&
+            isset($driverOptions['ssl_cert']) &&
+            isset($driverOptions['ssl_ca'])
```

Comment:
If we put ssl parameters as connection parameters against driver options will not run ok automatically in some php frameworks as Symfony.

Example:
 [Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]           
  Unrecognized options "ssl_key, ssl_cert, ssl_ca" under "doctrine.dbal.connections.bd1"

Are you sure do you prefer ssl parameters as connection parameters against driver options?